### ISSUE TITLE
Add option to set a custom HandBrakeCLI path

### ISSUE THREAD

**Issue by EpicKris** [75lb/handbrake-js#62]
To be able to pass our own HandBrake CLI Path as an option.

**Comment by 75lb** [comment]
Handbrake-js installs the `HandbrakeCLI` binary in the `node_modules/handbrake-js/bin` folder, so as a temporary solution you can override the installed bin using symlinks.. For example, on a mac with handbrake-js installed globally, this will override the installed `HandbrakeCLI` with a new version stored in my Downloads folder:

```
ln -s ~/Downloads/HandbrakeCLI /usr/local/lib/node_modules/handbrake-js/bin/HandbrakeCLI
```

**Comment by EpicKris** [comment]
That works as a temporary solution, but would still really like to see this included.

**Comment by 75lb** [comment]
Should this feature be added to the CLI, Node.js API or both? 

If we're adding it to the CLI, should we add it as an extra option? E.g. 

```
$ handbrake --input something.mov --output another.mp4 --HandbrakeCLI-path ./something/HandbrakeCLI
```

Or some other way? A config file?

**Comment by EpicKris** [comment]
I feel like this should be part of the API, the HandBrake CLI is the executable.

For example:
``` js
hbjs.spawn({
  input: 'something.avi',
  output: 'something.m4v',
  cliPath: './something/HandbrakeCLI'
});
```

**Comment by 75lb** [comment]
This is fixed on master, will post again once it is released..

**Comment by 75lb** [comment]
Released in [v6.0.0](https://github.com/75lb/handbrake-js/releases/tag/v6.0.0)

**Comment by 75lb** [comment]
Can also now [set HandbrakeCLI path via an environment variable](https://github.com/75lb/handbrake-js?tab=readme-ov-file#handbrakecli-path)
